packages/snjs/lib/services/api/mfa_service.ts

```diff
@@ -0,0 +1,169 @@
+import { Challenge, ChallengeKeyboardType, ChallengePrompt, ChallengeReason, ChallengeValidation } from '@Lib/challenges';
+import { ChallengeService, SNAlertService } from '..';
+import { PureService } from '../pure_service';
+import { SNApiService } from './api_service';
+import { PromptTitles, SessionStrings, SignInStrings } from './messages';
+import { HttpResponse, StatusCode } from './responses';
+
+export enum MfaStatus {
+  EnteredCode = 'mfa-entered-code',
+  NotRequired = 'mfa-not-required',
+  Ignored = 'mfa-ignored',
+
+  ErrorMissingCode = 'mfa-error-missing-code',
+  ErrorResponse = 'mfa-error-response',
+}
+
+export type MfaResultOld = {
```

Comment:
I would quantify what "Old" means. Either in terms of a specific API version, or a date, or perhaps type of server.

Reply:
(Not in terms of a comment btw, but as the name of the type itself)

Reply:
This shall be removed before undrafting this PR.

Reply:
removed, so resolving